packages/client/machines/appMusicPlaylistsMachine.ts

```diff
@@ -573,14 +572,33 @@ export function createAppMusicPlaylistsMachine({
                             ROOM_CREATION_ACKNOWLEDGEMENT: {
                                 target: 'waitingForRoomReadiness',
 
-                                actions: send((_, { state }) =>
-                                    appMusicPlaylistsModel.events.SPAWN_PLAYLIST_ACTOR_FROM_STATE(
-                                        {
-                                            roomID: state.roomID,
-                                            state,
-                                        },
+                                actions: [
+                                    send((_, { state }) =>
+                                        appMusicPlaylistsModel.events.SPAWN_PLAYLIST_ACTOR_FROM_STATE(
+                                            {
+                                                roomID: state.roomID,
+                                                state,
+                                            },
+                                        ),
                                     ),
```

Comment:
I was wondering if we should not use [`raise`](https://xstate.js.org/docs/guides/actions.html#raise-action) action instead of [`send`](https://xstate.js.org/docs/guides/actions.html#send-action) when we want to do that kind of thing 🤔

I never used it but I will dive deeper to understand when we should favour one solution over the other.

Reply:
:thinking: you're right this is a bit a priority